static/docs/user-guide/install/latest.md

```diff
@@ -0,0 +1,35 @@
+# Install latest version
```

Comment:
the latest version is usually means the latest releases version. It's still a confusing name. Why don't you like Development version?

Reply:
Fixed.

> Why don't you like Development version?

No particular reason, just seemed confusing.